src/components/task-step/all-steps.cjsx

```diff
@@ -37,6 +43,16 @@ Reading = React.createClass
       overlay.innerHTML = title
       img.parentElement.appendChild(overlay)
 
+  detectImgAspectRatio: ->
+    root = @getDOMNode()
+    for img in root.querySelectorAll('img')
+      # Wait until an image loads before trying to detect its dimensions
+      img.onload = ->
+        if @width > @height
+          @parentNode.classList.add('horizontal')
```

Comment:
Can you prefix the classes so they do not collide with content classes (like `ui-` or `tutor-`) and so it's clear they are being added dynamically by js?